pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -388,8 +388,8 @@ func validateRolloutStrategy(rolloutStrategy placementv1beta1.RolloutStrategy) e
 			if err != nil {
 				allErr = append(allErr, fmt.Errorf("maxUnavailable `%+v` is invalid: %w", rolloutStrategy.RollingUpdate.MaxUnavailable, err))
 			}
-			if value < 1 {
-				allErr = append(allErr, fmt.Errorf("maxUnavailable must be greater than or equal to 1, got `%+v`", rolloutStrategy.RollingUpdate.MaxUnavailable))
+			if value < 0 {
```

Comment:
update our api comment? should we get the approval from liqian for that api comment change?

Reply:
+1 on updating the api comment as well if it is different from the PR.